src/Invoice.php

```diff
@@ -234,7 +234,7 @@ protected function formatAmount($amount)
     public function view(array $data)
     {
         return View::make('cashier::receipt', array_merge(
-            $data, ['invoice' => $this, 'user' => $this->user]
+            $data, ['invoice' => $this, 'owner' => $this->owner]
```

Comment:
You'll need to still send `user` to the view as well to not break backwards compatibility.